frontend/src/WhatHappenedForm.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { Trans } from '@lingui/macro'
+import { Form, Field } from 'react-final-form'
+import { css } from 'react-emotion'
+import Button from '@govuk-react/button'
+import PropTypes from 'prop-types'
+
+const textArea = css`
+  width: 500pt;
+  height: 200pt;
+  font-size: 19pt;
+`
+
+const submitButton = css`
+  margin-top: 20pt;
+`
+
+const validate = () => {}
+
+export const WhatHappenedForm = ({ onSubmit }) => (
+  <Form
+    onSubmit={onSubmit}
+    validate={validate}
+    render={({ handleSubmit, pristine, invalid }) => (
+      <form onSubmit={handleSubmit}>
+        <div>
+          <label htmlFor="whatHappened">What Happened</label>
```

Comment:
So we didn't used to have this label. I assume without it we're damaging accessibility?